.gitignore

```diff
@@ -489,3 +489,4 @@ build*/
 /tools/archlinux/**/pkg/
 /tools/archlinux/**/valeronoi*/
 /tools/archlinux/**/*.pkg.tar
+.vscode/settings.json
```

Comment:
It's probably better to update the .gitignore using the link at the beginning of the file and adding "visualstudiocode": https://www.toptal.com/developers/gitignore/api/c,qt,c++,linux,macos,python,windows,clion+all,qtcreator,visualstudiocode